ironfish/src/wallet/walletdb/transactionValue.ts

```diff
@@ -41,6 +44,14 @@ export class TransactionValueEncoding implements IDatabaseEncoding<TransactionVa
       bw.writeU32(sequence)
     }
 
+    const assetCount = value.assetBalanceDeltas.size
+    bw.writeU32(assetCount)
```

Comment:
I see why you did this but it's clear this encoding this BufferSet.

```suggestion
    bw.writeU32(value.assetBalanceDeltas.size)
```